templates/config/scorecard/patches/olm.config.yaml

```diff
@@ -14,37 +14,37 @@
     entrypoint:
     - scorecard-test
     - olm-crds-have-validation
-    image: quay.io/operator-framework/scorecard-test:v1.3.0
+    image: quay.io/operator-framework/scorecard-test:v1.7.1
     labels:
       suite: olm
       test: olm-crds-have-validation-test
-- op: add
-  path: /stages/0/tests/-
-  value:
-    entrypoint:
-    - scorecard-test
-    - olm-crds-have-resources
-    image: quay.io/operator-framework/scorecard-test:v1.3.0
-    labels:
-      suite: olm
-      test: olm-crds-have-resources-test
+# - op: add
```

Comment:
@komish mind adding a quick comment here and below why these particular tests are commented out (basically include the text from your PR summary :) )

Reply:
@jaypipes Sure thing - done and squashed.